src/WorkflowCore/Services/BackgroundTasks/RunnablePoller.cs

```diff
@@ -54,7 +54,8 @@ private async void PollRunnables(object target)
                     try
                     {
                         _logger.LogInformation("Polling for runnable workflows");                        
-                        var runnables = await _persistenceStore.GetRunnableInstances(DateTime.Now);
+                        var runnables = (await _persistenceStore.GetRunnableInstances(DateTime.Now)).ToArray();
+                        _logger.LogInformation($"Polled {runnables.Length} runnable workflows");
```

Comment:
Probably would make more sense to just leave the log with the number of polled records...